javascript/javascript1/week3/speed.js

```diff
@@ -0,0 +1,12 @@
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  function notThisFunctionName(travelInformation){
+      const hoursOf =Math.trunc(travelInformation.destinationDistance / travelInformation.speed);
```

Comment:
Out of curiosity, why are you using the method `Math.trunc`?

Reply:
I changed it with Math.floor!